packages/forma-36-react-components/src/components/Tabs/Tabs.css

```diff
@@ -0,0 +1,54 @@
+@import 'settings/colors';
+@import 'settings/typography';
+@import 'settings/transitions';
+@import 'settings/dimensions';
+
+.Tabs {
+  display: flex;
+}
+
+.Tabs .Tab {
+  margin-right: var(--spacing-l);
+}
+
+.Tab {
+  white-space: nowrap;
+  color: var(--color-text-dark);
+  position: relative;
+  cursor: pointer;
+  padding: 0 var(--spacing-s);
+  height: 56px;
+  line-height: 56px;
+  font-size: var(--font-size-m);
+  font-family: var(--font-stack-primary);
+  font-weight: var(--font-weight-normal);
+  outline: none;
+  text-decoration: none;
+}
+
+.Tabs .Tab:last-child {
+  margin-right: 0;
+}
+
+.Tab__selected {
+  font-weight: var(--font-weight-demi-bold);
+}
+
+.Tab:before {
+  content: '';
+  position: absolute;
+  background: transparent;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 3px;
+}
+
+.Tab:hover:before {
+  background: #a7e2ff;
```

Comment:
don't we have a token for that? Isn't that primary blue?

Reply:
--color-primary

Reply:
or --color-blue-dark i guess instead of this value

Reply:
it's quite different actually, maybe I could use token and play with opacity